jina/proto/jina_pb2.py

```diff
@@ -6,6 +6,7 @@
 from google.protobuf import message as _message
```

Comment:
Black reformats the whole file. It should be excluded. Seems like the exclusion is not working.